developer/doc_development/doc_development.tex

```diff
@@ -0,0 +1,461 @@
+
+\href{https://github.com/openturns/openturns/blob/master/python/doc/theory/data_analysis/qqplot_graph.rst}{\texttt{python/doc/theory/data\_analysis/qqplot\_graph.rst}}
+
+There are great RST and Sphinx \href{https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html}{\alert{cheatsheet}} available,
+but here are the basics:
+
+\begin{itemize}
+    \item A page can be \alert{indexed} as \texttt{page\_name} by writing \texttt{.. \_page\_name:} at the top.
+    \item Because of this, that page can be \alert{linked} to from anywhere in the doc with \texttt{:ref:`page\_name`}
+    \item \alert{Titles} are underlined (the number of underlining characters must match the title).
+    \item \alert{Inline math} is \LaTeX{} code with \texttt{\$a=b\$} replaced with \texttt{:math:`a=b`}.
+    \item \alert{Display math} is \LaTeX{} code with \texttt{\$\$a=b\$\$} replaced with
+    
+    \texttt{.. math::}
+    
+    \texttt{~~~~a = b}
+    \item \alert{Macros} are defined in \href{https://github.com/openturns/openturns/blob/master/python/doc/math_notations.sty}{\texttt{python/doc/math\_notations.sty}}
+\end{itemize}
+
+\end{frame}
+
+
+\begin{frame}[fragile]{Adding a new theory page}
+Code of the main table of contents (\href{https://github.com/openturns/openturns/blob/master/python/doc/theory/theory.rst}{python/doc/theory/theory.rst}):
+
+\begin{lstlisting}
+.. _theory:
+
+======
+Theory
+======
+
+The theoretical documentation.
+This contains an in-depth description of all algorithms.
+
+.. toctree::
+    :maxdepth: 2
+
+    data_analysis/data_analysis.rst
+    probabilistic_modeling/probabilistic_modeling.rst
+    meta_modeling/meta_modeling.rst
+    reliability_sensitivity/reliability_sensitivity.rst
+    numerical_methods/numerical_methods.rst
+\end{lstlisting}
+Table of contents of subdirectories are essentially the same.
+\end{frame}
+
+
+
+%%%%%%%%%%%%%%%%%%%%%
+% Examples          %
+%%%%%%%%%%%%%%%%%%%%%
+\section{Examples}
+
+\againframe{tableofcontents}
+
+
+\begin{frame}{Looking up the source Python script of an example}
+
+The Examples section is produced by \href{https://sphinx-gallery.github.io/stable/index.html}{\alert{Sphinx-gallery}}, a Sphinx extension.
+
+Let us consider the \alert{Kriging with an isotropic covariance function} example:
+
+\url{https://openturns.github.io/openturns/master/auto_meta_modeling/kriging_metamodel/plot_kriging_isotropic.html}
+
+Example source files are \alert{Python scripts}.
+The URL is still linked to the the source folder structure,
+but the \texttt{auto\_} part is a Sphinx-gallery artifact. Its source file is:
+
+\href{https://github.com/openturns/openturns/blob/master/python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_isotropic.py}{\texttt{python/doc/examples/meta\_modeling/kriging\_metamodel/\\plot\_kriging\_isotropic.py}}
+
+Example Python scripts can be viewed as RST-based Jupyter notebook source files.
```

Comment:
actually that would be markdown

Reply:
I don't understand what you mean.

Reply:
look at the content of the notebook cells, you'll find markdown not rst